Test.C

```diff
@@ -581,3 +549,65 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
         return;
     }
 }
+/******************************************************************************
+* Function:    SellingStore
+* Description: Interactive store where player can sell equipment
+* Parameters:  playerPtr, Player, pointer to the player object
+*              equipmentStock, Equipment, array of all possible equipments
+* Return:      void
+******************************************************************************/
+
+void SellingStore(struct Player *playerPtr, struct Equipment equipmentStock[NUM_EQUIPMENT]) {
```

Comment:
No space after end of statement before curly brackets per the style guide
```suggestion
void SellingStore(struct Player *playerPtr, struct Equipment equipmentStock[NUM_EQUIPMENT]){
```